src/modules/caen/test/CaenBaseTest.cpp

```diff
@@ -250,10 +246,11 @@ TEST_F(CaenBaseTest, DataReceiveGoodMiracles) {
   Settings.ConfigFile = MIRACLES_CONFIG;
   Settings.CalibFile = MIRACLES_CALIB;
 
-  Settings.DetectorPort = 9000;
+  Settings.DetectorPort = 9216;
   Settings.UpdateIntervalSec = 0;
   Settings.DumpFilePrefix = "deleteme_";
-  CaenBaseStandIn Readout(Settings, ESSReadout::Parser::MIRACLES);
+  Caen::CaenBase Readout(Settings, ESSReadout::Parser::MIRACLES);
+  Readout.Counters = {};
```

Comment:
Probably don't need this line now

Reply:
Agree, not necessary, I will remove this line